cms/envs/common.py

```diff
@@ -326,6 +326,10 @@
 
     # Instead of AuthenticationMiddleware, we use a cache-backed version
     'cache_toolbox.middleware.CacheBackedAuthenticationMiddleware',
+    # Enable SessionAuthenticationMiddleware in order to invalidate
+    # user sessions after a password change.
+    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
```

Comment:
Why are we using an old middleware that is now only a BWC shim in django 1.8?
Why not move to whatever is the new way of handling this in django 1.8?